matrix-sdk-android/src/main/java/org/matrix/android/sdk/internal/session/content/UploadContentWorker.kt

```diff
@@ -48,10 +48,10 @@ import java.io.File
 import java.util.UUID
 import javax.inject.Inject
 
-private data class NewImageAttributes(
-        val newWidth: Int?,
-        val newHeight: Int?,
-        val newFileSize: Int
+private data class NewAttachmentAttributes(
+        val newWidth: Int? = 0,
+        val newHeight: Int? = 0,
```

Comment:
Would have defaulted those 2 val to `null`

Reply:
Done.